core/src/main/resources/defaults/blueprints/cdp-sdx-medium-ha.bp

```diff
@@ -0,0 +1,428 @@
+{
+  "tags": {
+    "shared_services_ready": true
+  },
+  "description": "CDP 1.0 SDX-HA template with Atlas, HMS, Ranger and other services they are dependent on. Services like HDFS, HBASE have HA",
+  "blueprint": {
+    "cdhVersion": "7.0.0",
+    "displayName": "ha-datalake",
+    "hostTemplates": [
+      {
+        "refName": "master",
+        "roleConfigGroupsRefNames": [
+          "hbase-REGIONSERVER-BASE",
+          "hbase-MASTER-BASE",
+          "hdfs-DATANODE-BASE",
+          "hdfs-FAILOVERCONTROLLER-BASE",
+          "hdfs-JOURNALNODE-BASE",
+          "hdfs-NAMENODE-1",
+          "hive-HIVEMETASTORE-BASE",
+          "solr-SOLR_SERVER-BASE",
+          "zookeeper-SERVER-BASE"
+        ]
+      },
+      {
+        "refName": "master2",
+        "roleConfigGroupsRefNames": [
+          "hdfs-BALANCER-BASE",
+          "hbase-REGIONSERVER-BASE",
+          "hbase-MASTER-BASE",
+          "hdfs-DATANODE-BASE",
+          "hdfs-FAILOVERCONTROLLER-BASE",
+          "hdfs-JOURNALNODE-BASE",
+          "hdfs-NAMENODE-2",
+          "zookeeper-SERVER-BASE"
+        ]
+      },
+      {
+        "refName": "alpha",
+        "roleConfigGroupsRefNames": [
+          "ranger-RANGER_ADMIN-BASE",
+          "ranger-RANGER_USERSYNC-BASE",
+          "atlas-ATLAS_SERVER-1",
+          "hdfs-DATANODE-BASE",
+          "knox-KNOX_GATEWAY-BASE",
+          "ranger-RANGER_TAGSYNC-BASE",
+          "kafka-KAFKA_BROKER-BASE",
+          "zookeeper-SERVER-BASE",
+          "hdfs-JOURNALNODE-BASE"
+        ]
+      },
+      {
+        "refName": "alpha2",
+        "roleConfigGroupsRefNames": [
+          "kafka-GATEWAY-BASE",
+          "atlas-ATLAS_SERVER-BASE",
+          "kafka-KAFKA_BROKER-BASE",
+          "zookeeper-SERVER-BASE"
+        ]
+      },
+      {
+        "refName": "beta",
+        "roleConfigGroupsRefNames": [
+          "hdfs-DATANODE-BASE",
+          "zookeeper-SERVER-BASE"
+        ]
+      },
+      {
+        "cardinality": 1,
+        "refName": "idbroker",
+        "roleConfigGroupsRefNames": [
+          "knox-IDBROKER-BASE"
+        ]
+      }
+    ],
+    "services": [
+      {
+        "refName": "atlas",
+        "serviceType": "ATLAS",
+        "roleConfigGroups": [
+          {
+            "refName": "atlas-ATLAS_SERVER-BASE",
+            "roleType": "ATLAS_SERVER",
+            "base": true,
+            "configs": [
+              {
+                "name": "atlas_kafka_bootstrap_servers",
+                "value": "{{{format-join host_groups.master format='%s:9092' }}}"
```

Comment:
You are right. It has to be corrected. 
If you see a configuration in the blueprint most likely it was added because we had to add to make it work.
Let me try again and get back.

Reply:
Simply ssh to one of the hosts where Atlas is deployed and check `/var/log/atlas/application.log`.

Reply:
ok

Reply:
These configurations are needed at this point as CM is throwing NullPointerException when it is not provided and cluster setup is failing.